packages/plugin-import-export/src/components/ExportSaveButton/index.tsx

```diff
@@ -17,13 +26,24 @@ export const ExportSaveButton: React.FC = () => {
     },
   } = useConfig()
 
-  const { getData } = useForm()
+  const { getData, setModified } = useForm()
+  const modified = useFormModified()
 
   const label = t('general:save')
 
   const handleDownload = async () => {
+    let timeoutId: null | ReturnType<typeof setTimeout> = null
```

Comment:
We should be consistent about caps for acronyms. For example: `findByID` is the preferred formatting.